faq.html

```diff
@@ -12,9 +12,9 @@
     faqs:
     - question: I’m an employer looking to hire some developers, how do I get in touch
         with you?
-      answer: Check out our <a href='/hire/'>Hire a Graduate</a> page with profile
+      answer: Check out our <a href="/hire/">Hire a Graduate</a> page with profile
```

Comment:
Double break in the structured data, so why don't we leave them as single quotes here and just ignore the frontmatter from linting. In fact, all frontmatter should be ignored from htmllint since it's not valid html

Reply:
also, we can turn the linting back on in .circleci/config.yml!